lib/members_page.rb

```diff
@@ -0,0 +1,69 @@
+#!/bin/env ruby
+# encoding: utf-8
+# frozen_string_literal: true
+require 'scraped'
+
+class MembersPage < Scraped::HTML
+  decorator Scraped::Response::Decorator::AbsoluteUrls
+
+  field :member_rows do
```

Comment:
if you're rewriting using `Scraped`, you should really be aiming to leave `.rubocop_todo.yml` empty, rather than relying on the things we were skipping as bad in the original scraper. This method in particular seems very complex, and should really be split up a bit.